connexion/handlers.py

```diff
@@ -41,3 +41,10 @@ def randomize_endpoint(self):
 
     def get_path_parameter_types(self):
         return {}
+
+    async def __call__(self, *args, **kwargs):
```

Comment:
The `ResolverErrorHandler` can be registered instead of an `Operation` when the resolver cannot find a view function for an operation. The old `Operation` classes were called via their `function()` method, while the new middleware `Operation` classes are callable with the ASGI interface themselves. Before this refactoring, the `RoutingMiddleware` was using a patched version of the old `Operation` classes, but now it has it's own `RoutingOperation` instead.

I added this here so the `ResolverErrorHandler` is compatible with both the old and middleware `Operations`, which are still both used in different places.